tests/Oryx.RuntimeImage.Tests/NodeImagesTest.cs

```diff
@@ -275,11 +313,115 @@ public async Task RunNodeAppUsingProcessJson_withDebugging(string nodeVersion)
                 runArgs: new[] { "-c", runAppScript },
                 assertAction: async () =>
                 {
-                    var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/json/list");
-                    Assert.Contains("devtoolsFrontendUrl", data);
+                    var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/");
+                    Assert.Contains("Say It Again", data);
                 },
                 dockerCli: _dockerCli);
 
         }
+
+        [Theory]
+        [MemberData(nameof(TestValueGenerator.GetNodeVersions_SupportDebugging), MemberType = typeof(TestValueGenerator))]
```

Comment:
are we adding app insights only to the versions that support debugging, or do the versions just coincide?

Reply:
for now those are the versions app insights also supports, so I reused them